packages/sysweb3-keyring/src/keyring-manager.ts

```diff
@@ -896,10 +896,10 @@ export const KeyringManager = (): IKeyringManager => {
 
     const { wallet: _wallet } = getDecryptedVault();
     const { length } = Object.values(_wallet.accounts);
-    const seed = await mnemonicToSeed(mnemonic);
+    const seed = await mnemonicToSeed(getDecryptedMnemonic());
```

Comment:
It would be more appropriate to fix the getDecryptedVault function if there is any issue with it them using a new function to serve as seed